src/_includes/partials/header.njk

```diff
@@ -1,11 +1,24 @@
 <header class="header-container" role="navigation" aria-label="main navigation">
 
-  <a class="header-item" href="/">Home</a>
-  <a class="header-item" href="/product-and-services">Products and Services</a>
-  <a class="header-item" href="/people">People</a>
+  <a class="header-item" href="/"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
```

Comment:
It would be nice to center align the icons. Tho I couldn't figure it out.
<img width="748" alt="Screenshot 2023-06-08 at 10 35 44 PM" src="https://github.com/zinc-collective/www.zinc.coop/assets/16140873/08b14040-9a6d-415b-8e34-c0ceb3ad4153">

Reply:
Instead of centering, if you apply the class `inline-block` to icons, they will flow in one line as part of the text, so they will effectively become sort of a "bullet point" for each menu item. (See https://tailwindcss.com/docs/display#block-and-inline for docs)

Reply:
yup that looks better. thanks.
<img width="945" alt="Screenshot 2023-06-09 at 6 33 59 PM" src="https://github.com/zinc-collective/www.zinc.coop/assets/16140873/fe64d22a-cd95-4889-b9d1-29d739833d00">